api/types.go

```diff
@@ -125,6 +125,8 @@ type Runner struct {
 	RopeFrequencyBase  float32 `json:"rope_frequency_base,omitempty"`
 	RopeFrequencyScale float32 `json:"rope_frequency_scale,omitempty"`
 	NumThread          int     `json:"num_thread,omitempty"`
+	GSize              int     `json:"g_size,omitempty"`
```

Comment:
@cognitivetech  Doesn't llama.cpp accept these params: `grp-attn-n` and `grp-attn-w` ? Not sure where you got `g_size` and `w_size` from.

```go
GSize int `json:"grp-attn-n,omitempty"`
WSize int `json:"grp-attn-w,omitempty"`
```
https://github.com/ggerganov/llama.cpp/issues/4886#issuecomment-1890465266